### ISSUE TITLE
Bug: Testimonials

### ISSUE THREAD

**Issue by Psinergy** [live-composer/live-composer-page-builder#744]
Since 1.3.2, clicking on the "Continue Reading" button on a testimonial takes you to the main page as it 404's on the testimonial link (we have it setup in .htaccess to redirect to main page if 404). Now, installing back to 1.3 everything works fine. Have resaved permalinks, force CSS, changing CSS Location, etc.

WP: 4.8

Example: https://test.psinergytech.com/client-reviews/

**Comment by lumberman** [comment]
@xdima92x please check this bug agains our latest code.

**Comment by xdima92x** [comment]
Hi @lumberman We don't have the bug in our latest code.

Hi @Psinergy The module 'Testimonials' don't have a button 'Continue Reading'. You using the module 'Post'. We will soon update Live Composer.

**Comment by Psinergy** [comment]
This bug is STILL happening in 1.3.3

I've also tried this as suggested by support:

1.  Open your website for editing in a private/incognito mode. Let me know if it's any different. - Yes, had already tried that. It gives the same issue.
2. Try to deactivate all the other plugins for a minute and see if LC works as expected? This way we can tell if any other plugins cause the issue. - Yes, had already tried that, and tried it again after updating just now to WP 4.8.1, still same issue.
3. Try to activate a default WP theme for a minute and see if LC works as expected? This way we can tell if there is a problem with your theme. - Just tried Twenty Seventeen and still same issue.

Module is POST.

Basically, if you go to one of the testimonials, like https://test.psinergytech.com/testimonial/hannah-linsk-1222017/ (one of the Testimonials that we have in the Testimonials thing), it 404's.

**Comment by dominicklee** [comment]
The bug is still happening in the latest version of Live Composer. This is very unprofessional software.

Our website has always listed out the "Testimonials" from our clients and now when people click "Read More", it redirects back to our homepage. It is as if the website is a lie.

Authors: Please fix this issue

**Comment by Psinergy** [comment]
Yes, it would be nice if they would figure out why this is doing this. It used to work fine, than after it stopped working with how we had been using it for years, they haven't seemed uninterested in repairing that aspect, so we've been left at keeping the old, out-of-date version of the plugin, and probably will not be using their software when we decide to redo our websites.

**Comment by dominicklee** [comment]
@Psinergy Exactly! They have broken something and don't want to fix it. They don't even keep sufficient documentation on each of their modules.

What I have personally done to fix this issue was install a plugin called **Custom Post Type UI** and then created a category called "Testimonial" or whatever you want to call it. Then I moved ALL my posts which used to be in the old "Testimonials (LC)" to my new post type, "Testimonial". The way to fix this issue is not to even use the post type made by Live Composer because honestly they have such issues.

After this, update your website pages with the modules to point to "Testimonial" instead of "Testimonials". Don't use "Testimonials" module, because I know what you are trying to do. You want it to be treated as a post, with "Read More" option.

With that said, I can now update my Live Composer plugin to the newest version. Although I think their support is bad and needs to improve, the software is still effective. I hope the authors take note of the trouble that users have to go through.

**Comment by FoldingTheme** [comment]
Hi there. The Testimonials, have a dedicated module. Please use the dedicated module. Was a mistake from our side to not remove from beginning the custom post from the post module.  This option will be removed on the next update to avoid this confusion.

Thanks for understanding.

**Comment by Psinergy** [comment]
Actually, the previous way that it was done was actually much nicer. If it wasn't... I wouldn't be asking for it to be back. Truncating testimonials, etc with how the new module does it looks stupid. We've had several compliments from clients about how the testimonials appear on our website, and a handful I've shown the other way, and they have all told me that they like the way we have it much better.

So... **why** is it soooooooooo hard to get the post functionality, or whatever it is... for testimonials? Obviously, it use to work... :/
